spec/hosts/nodes/bastion1.cp.lsst.org_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+#
+# note that this hosts has interfaces with an mtu of 9000
+#
+describe 'bastion1.cp.lsst.org', :site do
```

Comment:
There is no `hieradata/node/bastion1.cp.lsst.org.yaml` file in the repo. Is this intended to be a role test? If so, it should under `spec/hosts/roles/`

Reply:
yeap, you're right, not needed